src/app/shared/components/personal-achievements/personal-achievements.component.ts

```diff
@@ -1,27 +1,28 @@
+import { RequestDataService } from './../../services/request-data.service';
 import { Component, Input, OnInit } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
 
-import { RequestDialogComponent } from 'src/app/dialogs/request-dialog/request-dialog.component';
-
-import { AuthUserService } from '../../services/auth-user.service';
+import { RequestDialogComponent } from 'src/app/shared/dialogs/request-dialog/request-dialog.component';
 
 import { Achievement } from '../../models/achievement';
+import { Observable } from 'rxjs';
 
 @Component({
   selector: 'app-personal-achievements',
   templateUrl: './personal-achievements.component.html',
   styleUrls: ['./personal-achievements.component.scss']
 })
 export class PersonalAchievementsComponent implements OnInit {
-  @Input() title: string = '';
+  personalAchievements$: Observable<Achievement[]>;
 
-  personalAchievements: Achievement[];
+  @Input() title: string = '';
+  @Input() achievementsCount: number;
```

Comment:
@Input/@Output/@... goes first. After them we provide properties. Please fix this